requirements.txt

```diff
@@ -5,3 +5,4 @@ netCDF4>=1.6.4
 requests
 pytz
 simplekml
+dill
```

Comment:
have you searched about this lib before adding it to the requirements? is this stable enough?

Also, you must edit the pyproject.toml file otherwise this change is useless.

Reply:
The `pyproject` file imports the default requirements one, so I don't think I need to change there. This library seems to be stable enough, they release about every six months consistently and have a large userbase / star count.